maui-toolkit/TabView/Selection-Indicator-Customization.md

```diff
@@ -223,4 +223,22 @@ tabView.IndicatorCornerRadius  = 5;
 
 The following image shows the selection indicator with the corner radius:
 
-![.NET MAUI TabView Selection Indicator Corner Radius](images/IndicatorCornerRadius.png) 
\ No newline at end of file
+![.NET MAUI TabView Selection Indicator Corner Radius](images/IndicatorCornerRadius.png) 
+
+## Indicator's Stroke Thickness
+You can customize the stroke thickness of the selection indicator using the [IndicatorStrokeThickness](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TabView.SfTabView.html#Syncfusion_Maui_TabView_SfTabView_IndicatorStrokeThickness) property in the Tab View.
```

Comment:
Remove this API link, this is not the correct link.

Reply:
Removed this API link.